playbooks/roles/deploy-osh/tasks/main.yml

```diff
@@ -130,17 +130,43 @@
     - horizon
     - run
 
+- name: Install openstack client
+  package:
+    name: python-openstackclient
+    state: present
+  register: _pkginstall
```

Comment:
do we really need all this stuff with retries? This should only be used if we expect issues but why would we expect issues on a simple package install?

Reply:
usually not, but it could theoretically happen that some zypper process is already running...

Reply:
So I'm just using same values I've found elsewhere

Reply:
shouldnt then we use the zypper module with `environment:
    ZYPP_LOCK_TIMEOUT: 20` as options to suse-ify it more??

Reply:
substitute 20 with the wanted value of course :)

Reply:
Might be good idea to do extra PR that would replace all package installations in our plays

Reply:
> do we really need all this stuff with retries? This should only be used if we expect issues but why would we expect issues on a simple package install?

network related issues. That's why we brought that into ansible-lint.
ansible-lint will complain if you don't bring that.

Let's not make it extra hard with locks, just make it retries, so we don't have to change that everywhere. If we need to change the module to have this ZYPP_LOCK_TIMEOUT by default, we can too.